components/gitpod-protocol/src/workspace-instance.ts

```diff
@@ -324,3 +327,13 @@ export interface ImageBuildLogInfo {
     url: string;
     headers: { [key: string]: string };
 }
+
+/**
+ * Holds metrics about the workspace instance
+ */
+export interface WorkspaceInstanceMetrics {
+    image?: Partial<{
+        totalSize: number;
```

Comment:
Comments would be :1st_place_medal:

Reply:
Added in [`b8de461` (#20426)](https://github.com/gitpod-io/gitpod/pull/20426/commits/b8de4614b3b2ab155c408c4b62f2208079532aef) 🙏